tests/cucumber/features/channels/channel_creation.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Channel creation
+  As an authenticated user
+  I want to create a channel
+
+  Background:
+    Given I am on the home page
+    And I am signed in
+    When I click on the new channels button
+
+  # Test creation and validation of new channels
+  @dev
+  Scenario: Create a valid new channel
+    When I enter "Galaxy" into the title field
+    And I enter "pony+galaxy" into the query field
+    And I submit the new channel form
+    Then channel "title" should display "Galaxy"
+    And channel "query" should display "pony+galaxy"
```

Comment:
Could you use a regex that just tests that the route is of the form `/channels/[a-z0-9]*/` ? It doesn't need to know the exact route, it can just test that the route *changed*.